build/device.mk

```diff
@@ -272,6 +272,16 @@ PRODUCT_COPY_FILES += \
     vendor/xen/sensors/cfg/sensors-config.json:$(TARGET_COPY_OUT_VENDOR)/etc/vehicle/sensors-config.json
 
 # Exterior View System (EVS)
+USE_CARLA_EVS=true
```

Comment:
Hi @DKlimenko, maybe you should use the "USE_CARLA_EVS?=true" to allow overwriting this value?

Otherwise, what is the reason for having a condition at line 276 for the variable, which is always equal to true?

Reply:
It is our flag. Requested by Andrey

Reply:
Hi @DKlimenko, but shouldn't it be used with "?=". Could you, please, explain why it is needed? What is the reason for it?

Reply:
Hi @svlad-90, it is done for future compile flag integration for EVS functionality

Reply:
This flag should be set externally by moulin or shell. Since it is a generic code and we at least now don't want to enable it all the time.

Reply:
As was agreed with @arminn , flag was removed.